src/main/java/org/isf/lab/manager/LabManager.java

```diff
@@ -79,8 +79,8 @@ protected void setPatientConsistency(Laboratory laboratory) {
 	 */
 	protected void validateLaboratory(Laboratory laboratory) throws OHDataValidationException {
 		List<OHExceptionMessage> errors = new ArrayList<>();
-		if (laboratory.getExamDate() == null) {
-			laboratory.setExamDate(LocalDate.now());
+		if (laboratory.getCreatedDate() == null) {
+			laboratory.setCreatedDate(TimeTools.getNow());
```

Comment:
This should be managed by JPA automatically

Reply:
where `setCreatedDate` is defined?

Reply:
Use you are correct it can be removed.

`setCreateDate` is found in `org.isf.utils.db.Auditable`

Reply:
is this removed or not yet?